lib/utility.js

```diff
@@ -58,7 +58,7 @@ module.exports = {
     }
   },
 
-  sendEmail(email, mailMessage) {
+  sendEmailForDev(email, mailMessage) {
```

Comment:
I think the 'sendEmailForDev' variable name could better reflect it's purpose. What do you think?

Reply:
Noted